docs/tables/github_repository_sbom.md

```diff
@@ -0,0 +1,22 @@
+# Table: github_repository_sbom
+
+The `github_repository_sbom` table can be used to query information about packages listed in the SBOM of a repository.
+
+**You must specify which repository** in the where or join clause using the `repository_full_name` column.
+
+## Examples
+
+### List SBOM packages
+
+```sql
+select
+  github_repository_sbom,
```

Comment:
Does this example work? I don't think having the table name in the `select` statement would yield anything.

I'd suggest adding some basic SBOM columns like `spdx_id` and `spdx_version` instead.